index.html

```diff
@@ -6,49 +6,208 @@
+                    <input type="button" value="Ruby">
+                </div>
+                <div class="projects-details">
+                    <input type="button" value="See project">
+                </div>
+            </div>
+            <div class="img-five remaining-img">
+                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                    industry's standard.
+                </p>
+                <div class="skills-details">
+                    <input type="button" value="HTML">
+                    <input type="button" value="Bootstrap">
+                    <input type="button" value="Ruby">
+                </div>
+                <div class="projects-details">
+                    <input type="button" value="See project">
+                </div>
+            </div>
+            <div class="img-six remaining-img">
+                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                    industry's standard.
+                </p>
+                <div class="skills-details">
+                    <input type="button" value="HTML">
+                    <input type="button" value="Bootstrap">
+                    <input type="button" value="Ruby">
+                </div>
+                <div class="projects-details">
+                    <input type="button" value="See project">
+                </div>
+            </div>
+            <div class="img-seven remaining-img">
+                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                    industry's standard.
+                </p>
+                <div class="skills-details">
+                    <input type="button" value="HTML">
+                    <input type="button" value="Bootstrap">
+                    <input type="button" value="Ruby">
+                </div>
+                <div class="projects-details">
+                    <input type="button" value="See project">
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- ----------------------About section----------------->
+    <section class="about-section">
+        <div class="about-introduction">
+            <div class="about-introduction-content">
+                <h1 class="about-heading">About me</h1>
+                <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some
+                    of my work and experience! If you like what you see and have a project you need coded, don't
+                    hesitate to contact me.
+                </p>
```

Comment:
- [ ] kindly consider giving this paragraph the same font family as the one provided in the Figma design this will make your project an exact duplicate of the provided design